### PR TITLE
Add Reason icon

### PR DESCRIPTION
**Pull request by wyze**
**Changes proposed:**
* [X] Add
* [ ] Prepare
* [ ] Delete
* [ ] Fix

**Things I've done:**
* [X] Extension builds
* [X] Tests pass

**Preview**

screen shot 2017-03-23 at 8 27 41 am

**Notes**

I know SVG is preferred but they currently don't have an SVG version of their logo. I guess it wouldn't be too difficult to make one though.

### CODE REVIEW

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -262,6 +262,7 @@ export const extensions: IFileCollection = {
     { icon: 'reactjs', extensions: [], languages: [languages.javascriptreact], format: FileFormat.svg },
     { icon: 'reacttemplate', extensions: ['rt'], format: FileFormat.svg },
     { icon: 'reactts', extensions: [], languages: [languages.typescriptreact], format: FileFormat.svg },
+    { icon: 'reason', extensions: ['re', 'rei'], languages: [languages.reason], format: FileFormat.png },
```
Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: Remove the `extensions` references. Those are provided by the extension itself (https://github.com/freebroccolo/vscode-reasonml/blob/master/package.json#L181).

Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: Reminder: Replace the `format` once an SVG icon is provided.

Comment (wyze) [inline src/icon-manifest/supportedExtensions.ts]: Ah, neat that it pulls extensions from the `vscode-reasonml` extension.

**Comment by JimiC** [comment]
@wyze I've pushed to your repo an SVG version of the icon. Also fixed the `extension` reference. So we are good.

**Pull request merged**